Nodejs/Product/Nodejs/NodejsConstants.cs

```diff
@@ -44,7 +44,12 @@ internal static class NodejsConstants
 
         public const ushort DefaultDebuggerPort = 5858;
 
+        public const string CompileItemType = "Compile";
```

Comment:
It would be nice to clean these up if it turns out we can update the targets file instead.

Reply:
Yeah, I'll maker sure to clean up this constants if when they are not used anymore.